pkg/ccl/sqlccl/backup_test.go

```diff
@@ -162,13 +215,25 @@ func backupAndRestore(
 
 		var unused string
 		var dataSize int64
+		before := timeutil.Now()
 		sqlDB.QueryRow(fmt.Sprintf(`BACKUP DATABASE bench TO '%s'`, dest)).Scan(
 			&unused, &unused, &unused, &dataSize,
 		)
 		approxDataSize := int64(backupRestoreRowPayloadSize) * numAccounts
 		if max := approxDataSize * 2; dataSize < approxDataSize || dataSize > max {
 			t.Errorf("expected data size in [%d,%d] but was %d", approxDataSize, max, dataSize)
 		}
+
+		verifyJob(t, sqlDB, before, sql.JobStatusSucceeded, &sql.JobPayload{
+			Creator:     security.RootUser,
+			Description: fmt.Sprintf("BACKUP DATABASE bench TO '%s'", dest),
+			DescriptorIDs: []sqlbase.ID{
+				keys.DescriptorTableID,
+				keys.UsersTableID,
+				keys.MaxReservedDescID + 1 + 1,
```

Comment:
Is this the correct list of descriptors? (Ditto for the restore below, which is missing the descriptor and users table.)